games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
+	
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0;
```

Comment:
ah ok, so $destdir can be used also :) (overlooked that one) 
the iom file isn't used by haikuporter, but could be handy if someone else in the future would try to enhance or change it (if needed) :)